src/zenml/entrypoints/step_entrypoint_configuration.py

```diff
@@ -51,6 +51,20 @@
 MATERIALIZER_SOURCES_OPTION = "materializer_sources"
 
 
+def _b64_encode(input_: str) -> str:
```

Comment:
This function seems to be something we can put in some basic utils as its helpful for many things

Reply:
Agreed, i'll put it into `string_utils`